developmentScripts/default.ps1

```diff
@@ -0,0 +1,71 @@
+FormatTaskName "------- Executing Task: {0} -------"
+Framework "4.6" #.NET framework version
+
+properties {
+    $build_config = "Debug"
+    $verbosity = "normal" # q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]
+    $build_artifacts = Join-Path $root "Artifacts"
+    $test_logs = Join-Path $build_artifacts "TestLogs"
+    $build_logs = Join-Path $build_artifacts "BuildLogs"
+    $solution = Join-Path $root "..\Gravity\Gravity.sln"  
+}
+
+task default -Depends LocalBuild
+task LocalBuild -Depends Compile, IntegrationTest
+
+task NuGetRestore -Description "Restore NuGet packages for the solution" {
+    Write-Verbose "Solution :  $solution"
+    exec { & $nuget_exe @('restore', $solution) }
+}
+
+task CompileInitialize -Description "Cleanup pre-existing build artifacts, if they exist" {
+    InitializeDirectory $build_logs
+}
+
+task TestInitialize -Description "Cleanup pre-existing test artifacts, if they exist" {
+    InitializeDirectory $test_logs
+}
+
+task Compile -Depends CompileInitialize, NuGetRestore -Description "Compile the solution" {
+    Write-Verbose "Configuration: $build_config"
+    Write-Verbose "Verbosity: $verbosity"
+
+    # https://msdn.microsoft.com/en-us/library/ms164311.aspx
+    exec { msbuild @($solution,
```

Comment:
This throws an error if using the VS2015 MSBuild.  Looks like Psake uses a setting in the registry HKEY_LOCAL_MACHINE -> SOFTWARE -->Microsoft --> MSBuild -->ToolsVersion --> 14.0.  I updated the MSBuildToolsPath to C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\ and it works.  I will create an issue for this, but I think we should update the error message if possible to say that changing this key may get it working.

Reply:
I have added this bit to the script so a user knows to update the MSBUILD version in the registry key. I also found that installing a lower version of nuget makes psake pick MSBUILD for VS 2015. I have added this bit to the build.ps1 script